public/images/red-list.svg

```diff
@@ -0,0 +1,69 @@
+<svg version="1.1" id="Area_without_line" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
```

Comment:
Just wondering: is this file formatted as it should?
I mean, the indentation looks off, so I'm not sure it's was originally like this or it's something to do with copy-paste

Reply:
Thanks!